src/views/Dashboard.vue

```diff
@@ -276,7 +276,8 @@ export default {
 }
 </script>
 
-<style lang="scss" scoped>
+<style lang="scss" >
+@import "@/assets/styles/theme.scss";
```

Comment:
Why did you unscope this? This component should be able to see all the assets already.

Also you don't need the theme import since you don't use any of the variables in it.

Also you don't need the blank line 299.